src/components/v5/common/CreateUserForm/CreateUserFormInputs.tsx

```diff
@@ -11,18 +11,30 @@ const displayName = 'common.CreateUserFormInputs';
 const CreateUserFormInputs = () => {
   const {
     register,
-    formState: { errors, isSubmitting, dirtyFields },
+    formState: { errors, isSubmitting, dirtyFields, submitCount },
   } = useFormContext();
   const emailAddressError = errors.emailAddress?.message as string | undefined;
+  const showEmailAddressError = Boolean(
```

Comment:
If this is a recurring pattern, might be interesting to factor this out into a helper (doesn't need to be a hook though, we have so many of those).